### PR TITLE
update vc json schema

### PR DESCRIPTION
**Pull request by decentralgabe**
as per https://github.com/w3c-ccg/vc-json-schemas/pull/112

### CODE REVIEW

File: schema/jsonschema.go [hunk 1]
```diff
@@ -20,8 +24,9 @@ var (
 )
 
 func init() {
-	jsonschema.Loaders["http"] = httploader.Load
-	jsonschema.Loaders["https"] = httploader.Load
+	httploader.Client = &http.Client{
+		Timeout: time.Second * 10,
+	}
```
Comment (andresuribe87) [inline schema/jsonschema.go:29]: Not sure I follow why this changes? Mind elaborating?

**Review by andresuribe87 — APPROVED** [review]

File: schema/jsonschema.go (hunk 1 above)
Comment (decentralgabe) [reply schema/jsonschema.go:29]: the default client does not have a timeout configured https://github.com/santhosh-tekuri/jsonschema/issues/92#issuecomment-1309794888
